### PR TITLE
Refactor dsl envvar

### PR DESCRIPTION
**Pull request by rapp-fzi**
Please check if envvalue lookup from DSL is ok

### CODE REVIEW

File: tests/org.palladiosimulator.simexp.dsl.smodel.interpreter.tests/src/org/palladiosimulator/simexp/dsl/smodel/interpreter/util/EnvironmentalDynamicsCreatorHelper.java [hunk 1]
```diff
@@ -0,0 +1,190 @@
+package org.palladiosimulator.simexp.dsl.smodel.interpreter.util;
```
Comment (rsfzi) [inline tests/org.palladiosimulator.simexp.dsl.smodel.interpreter.tests/src/org/palladiosimulator/simexp/dsl/smodel/interpreter/util/EnvironmentalDynamicsCreatorHelper.java:1]: Should this class not rather reside in the test fragment?

File: tests/org.palladiosimulator.simexp.dsl.smodel.interpreter.tests/src/org/palladiosimulator/simexp/dsl/smodel/interpreter/util/EnvironmentalDynamicsTestModels.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package org.palladiosimulator.simexp.dsl.smodel.interpreter.util;
```
Comment (rsfzi) [inline tests/org.palladiosimulator.simexp.dsl.smodel.interpreter.tests/src/org/palladiosimulator/simexp/dsl/smodel/interpreter/util/EnvironmentalDynamicsTestModels.java:1]: Should this class not rather reside in the test fragment?

File: tests/org.palladiosimulator.simexp.dsl.smodel.interpreter.tests/src/org/palladiosimulator/simexp/dsl/smodel/interpreter/util/PcmModelsCreatorHelper.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package org.palladiosimulator.simexp.dsl.smodel.interpreter.util;
```
Comment (rsfzi) [inline tests/org.palladiosimulator.simexp.dsl.smodel.interpreter.tests/src/org/palladiosimulator/simexp/dsl/smodel/interpreter/util/PcmModelsCreatorHelper.java:1]: Should this class not rather reside in the test fragment?

File: tests/org.palladiosimulator.simexp.dsl.smodel.interpreter.tests/src/org/palladiosimulator/simexp/dsl/smodel/interpreter/util/ProbabilityDistributionCreatorHelper.java [hunk 4]
```diff
@@ -0,0 +1,134 @@
+package org.palladiosimulator.simexp.dsl.smodel.interpreter.util;
```
Comment (rsfzi) [inline tests/org.palladiosimulator.simexp.dsl.smodel.interpreter.tests/src/org/palladiosimulator/simexp/dsl/smodel/interpreter/util/ProbabilityDistributionCreatorHelper.java:1]: Should this class not rather reside in the test fragment?

File: tests/org.palladiosimulator.simexp.dsl.smodel.interpreter.tests/src/org/palladiosimulator/simexp/dsl/smodel/interpreter/util/SModelCreatorHelper.java [hunk 5]
```diff
@@ -0,0 +1,34 @@
+package org.palladiosimulator.simexp.dsl.smodel.interpreter.util;
```
Comment (rsfzi) [inline tests/org.palladiosimulator.simexp.dsl.smodel.interpreter.tests/src/org/palladiosimulator/simexp/dsl/smodel/interpreter/util/SModelCreatorHelper.java:1]: Should this class not rather reside in the test fragment?

**Review by rsfzi — APPROVED** [review]

File: tests/org.palladiosimulator.simexp.dsl.smodel.interpreter.tests/src/org/palladiosimulator/simexp/dsl/smodel/interpreter/util/SModelCreatorHelper.java (hunk 5 above)
Comment (rapp-fzi) [reply tests/org.palladiosimulator.simexp.dsl.smodel.interpreter.tests/src/org/palladiosimulator/simexp/dsl/smodel/interpreter/util/SModelCreatorHelper.java:1]: Isn't org.palladiosimulator.simexp.dsl.smodel.interpreter.tests the test fragment?

File: tests/org.palladiosimulator.simexp.dsl.smodel.interpreter.tests/src/org/palladiosimulator/simexp/dsl/smodel/interpreter/util/ProbabilityDistributionCreatorHelper.java (hunk 4 above)
Comment (rapp-fzi) [reply tests/org.palladiosimulator.simexp.dsl.smodel.interpreter.tests/src/org/palladiosimulator/simexp/dsl/smodel/interpreter/util/ProbabilityDistributionCreatorHelper.java:1]: Isn't org.palladiosimulator.simexp.dsl.smodel.interpreter.tests the test fragment?

File: tests/org.palladiosimulator.simexp.dsl.smodel.interpreter.tests/src/org/palladiosimulator/simexp/dsl/smodel/interpreter/util/PcmModelsCreatorHelper.java (hunk 3 above)
Comment (rapp-fzi) [reply tests/org.palladiosimulator.simexp.dsl.smodel.interpreter.tests/src/org/palladiosimulator/simexp/dsl/smodel/interpreter/util/PcmModelsCreatorHelper.java:1]: Isn't org.palladiosimulator.simexp.dsl.smodel.interpreter.tests the test fragment?

Comment (rsfzi) [reply tests/org.palladiosimulator.simexp.dsl.smodel.interpreter.tests/src/org/palladiosimulator/simexp/dsl/smodel/interpreter/util/PcmModelsCreatorHelper.java:1]: yes

Comment (rsfzi) [reply tests/org.palladiosimulator.simexp.dsl.smodel.interpreter.tests/src/org/palladiosimulator/simexp/dsl/smodel/interpreter/util/PcmModelsCreatorHelper.java:1]: But I can see only the package...

**Pull request merged**